src/components/EntryForm.js

```diff
@@ -195,32 +200,32 @@ const EntryForm = ({
         </div>
 
         <fieldset className={styles.feeGroup}>
-          <legend className={helpers.visuallyHidden}>Fees</legend>
+          <legend className={helpers.visuallyHidden}>Payment</legend>
 
           <label>
             This toilet is free
             <input
               ref={register}
               name="isFree"
               type="checkbox"
-              checked={isFree}
+              checked={!isPaymentRequired}
               className={styles.feeToggle}
-              onChange={(event) => setIsFree(event.target.checked)}
+              onChange={(event) => setIsPaymentRequired(event.target.checked)}
```

Comment:
I think `event.target.checked` needs to be inverted here.
If the checkbox is checked then we should set `isPaymentRequired` to `false` instead of `true`

Reply:
I think so too.

I got tangled up in booleans